recipes/ngspicesimserver/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "NgspiceSimServer" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NyanCAD/NgspiceSimServer/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e293ba5ae835826781a0351b7f4df051731ba8e95864803da331b81fc8206766
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - capnproto
+    - ngspice-lib
+    - openssl 1.1*
```

Comment:
Turns out we do need the `openssl 1.1*` somewhere otherwise it complains. I stole that from here: https://github.com/conda-forge/capnproto-feedstock/blob/12e724ce8e645c1ce1aed3e4d24288c9558bb360/recipe/meta.yaml#L21

Not 100% clear on the host/build distinction.